### ISSUE TITLE
iOS not builds in Ionic (Error code 65)

### ISSUE THREAD

**Issue by AlexDZI** [FortuneN/cordova-plugin-zeep#15]
** BUILD FAILED **

The following build commands failed:
	CompileC /Users/user13/Library/Developer/Xcode/DerivedData/RunAudioFX-gxtuhbgatrmuldfktgoiriftpbnn/Build/Intermediates.noindex/RunAudioFX.build/Debug-iphonesimulator/RunAudioFX.build/Objects-normal/x86_64/FiNeZeep.o RunAudioFX/Plugins/cordova-plugin-zeep/FiNeZeep.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
Error: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/user13/Desktop/runAudioFX-A/runaudiofx/platforms/ios/cordova/build-debug.xcconfig,-workspace,RunAudioFX.xcworkspace,-scheme,RunAudioFX,-configuration,Debug,-sdk,iphonesimulator,-destination,platform=iOS Simulator,name=iPhone X,build,CONFIGURATION_BUILD_DIR=/Users/user13/Desktop/runAudioFX-A/runaudiofx/platforms/ios/build/emulator,SHARED_PRECOMPS_DIR=/Users/user13/Desktop/runAudioFX-A/runaudiofx/platforms/ios/build/sharedpch

[ERROR] An error occurred while running cordova build ios (exit code 1).

Plugins:
cordova-clipboard 1.1.1 "Clipboard"
cordova-plugin-camera 4.0.1 "Camera"
cordova-plugin-device 1.1.4 "Device"
cordova-plugin-file 6.0.1 "File"
cordova-plugin-file-transfer 1.7.1 "File Transfer"
cordova-plugin-filepath 1.0.2 "FilePath"
cordova-plugin-filepicker 1.1.4 "File Picker"
cordova-plugin-inappbrowser 2.0.2 "InAppBrowser"
cordova-plugin-ionic 4.1.3 "IonicCordova"
cordova-plugin-ionic-webview 1.1.16 "cordova-plugin-ionic-webview"
cordova-plugin-media 5.0.2 "Media"
cordova-plugin-nativeaudio 3.0.9 "Cordova Native Audio"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-whitelist 1.3.1 "Whitelist"
cordova-plugin-x-socialsharing 5.2.1 "SocialSharing"
cordova-plugin-zeep 0.0.4 "Zeep"
es6-promise-plugin 4.1.0 "Promise"
ionic-plugin-keyboard 2.2.1 "Keyboard"

Can you help, please? Maybe this related with cordova-plugin-ionic?

**Comment by jondediego** [comment]
I am having the same issue. Did you solve this?

**Comment by AlexDZI** [comment]
> I am having the same issue. Did you solve this?

Yes. Conflict was in plugins **cordova-plugin-ionic** and **cordova-plugin-zeep**.
Ionic plugin use zip libraries, which conflict with zip libraries from zeep plugin, so i was remove zeep plugin from project.

**Comment by jondediego** [comment]
I solved it forcing removal of cordova-plugin-zeep and adding it again.
ionic cordova plugin remove cordova-plugin-zeep —force
ionic cordova plugin add cordova-plugin-zeep
